app/grants/management/commands/ingest_grant_txns.py

```diff
@@ -31,25 +31,30 @@
 from economy.tx import headers
 from economy.utils import convert_token_to_usdt
 from grants.models import Contribution, Grant, Subscription
+from perftools.models import JSONStore
 from web3 import Web3
 
-"""
-NOTE: This script only supports (1) regular L1 BulkCheckout, and (2) zkSync with L1 deposit. The
-third case, pure zkSync donations, is not yet supported because none of the missing contributions
-(at the time of this writing) are from this case.
-"""
-
 # ============================== Set the variables for this run here ===============================
-network = "rinkeby" # "mainnet" or "rinkeby"
-handle = "mds1" # Gitcoin username of user who made the contributions
-txid = "0xdc85d562b0c0caf25e03e692d0ccc696ebcf5046cb36d12d4fdf88ceb7cac0e1" # L1 transaction hash
-token = "USDT" # token contributions were made in
-to_address = "0xaBEA9132b05A70803a4E85094fD0e1800777fBEF" # recipient of the L1 transaction
-from_address = "0x60A5dcB2fC804874883b797f37CbF1b0582ac2dD" # from address for hash given by txid 
-do_write = True # Use True to save contributions to Database
-created_on = datetime.datetime(2020, 10, 5, 17, 22, tzinfo=pytz.UTC) # UTC timestamp oftxid 
-gitcoin_zksync_addr = "0x5b4e39e6649a9c0afd39f068f7076f0ea3125e8a" # user's Gitcoin zkSync address, only used for zkSync cases
 
+# network should be "mainnet" or "rinkeby"
+network = "rinkeby"
```

Comment:
not sure why we're defaulting network to rinkeby here?

Reply:
The defaults here don't really matter much! They're mainly just to serve as a template. But generally I like using rinkeby as a default for manual actions like this to avoid accidentally doing something on mainnet